### PR TITLE
Add display validation to dev

### PR DESCRIPTION
**Pull request by colinking**
## Description

Matches the updated validation in https://github.com/airplanedev/airport/pull/3530

Resolves TSK-800

## Test plan

Run a task locally that produced a large markdown doc. Ensure an error is returned.

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #779** <a href="https://app.graphite.dev/github/pr/airplanedev/cli/779" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/cli/779?utm_source=stack-comment).

File: pkg/server/apiext/apiext.go [hunk 1]
```diff
@@ -370,9 +370,24 @@ func CreateDisplayHandler(ctx context.Context, state *state.State, r *http.Reque
...
+
+		maxMarkdownLength := 100000
+		if len(display.Content) > maxMarkdownLength {
+			return CreateDisplayResponse{}, errors.Errorf("content too long: expected at most %d characters, got %d", maxMarkdownLength, len(display.Content))
+		}
 	case "table":
 		display.Rows = req.Display.Rows
 		display.Columns = req.Display.Columns
+
+		maxRows := 10000
```
Comment (ameyakhare) [inline pkg/server/apiext/apiext.go:382]: I feel like these numbers should be from the displays package instead of copied here

Comment (ameyakhare) [reply pkg/server/apiext/apiext.go:382]: or like we should write some sort of validation in the displays package and then call that validation here

**Review by ameyakhare — APPROVED** [review]

File: pkg/server/apiext/apiext.go (hunk 1 above)
Comment (colinking) [reply pkg/server/apiext/apiext.go:382]: Agreed, but that would require us to extract displays from `airport`. Not going to do that now.

**Pull request merged**